ui/articles-aem-renderer/build.gradle

```diff
@@ -0,0 +1,44 @@
+apply plugin: 'com.jakewharton.butterknife'
+
+android {
+    defaultConfig {
+        vectorDrawables.useSupportLibrary = true
```

Comment:
I believe this is already taken care of in the root build.gradle file

Reply:
This was in the Trac-Render gradle file so that is why I added it.  I can remove it if you think it is unnecessary.

Reply:
let's remove it for the moment until we actually get to the point of adding views and vector images to this module